lib/tasks/institution_uuid.rake

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+namespace :degrees do
+  desc <<~DESC
+    add institution_uuid to degrees
+  DESC
+
+  task add_institution_uuid: :environment do
+    degrees = Degree.uk
+
+    def hesa_code_for_institute(degree)
+      Hesa::CodeSets::Institutions::MAPPING.filter_map do |institute|
+        if institute.include?(degree.institution)
+          institute.first.to_i.to_s
+        end
+      end
+    end
+
+    degrees.find_each do |degree|
+      hesa_code_for_institute = hesa_code_for_institute(degree)
+      next if hesa_code_for_institute.blank?
+
+      institutions = DfE::ReferenceData::Degrees::INSTITUTIONS.some({ hesa_itt_code: hesa_code_for_institute.first })
+      degree.update(institution_uuid: institutions&.first&.id)
```

Comment:
Should we use `update_column` instead so the change doesn't appear in the timeline?

Reply:
that's a good point! Makes sense. I don't think we'd want to update all the updated_at on so many degrees for this change so will change to `update_column` instead